### PR TITLE
fix :: project name clean before saving to DB

### PR DESCRIPTION
**Pull request by sagarsangwan**

### CODE REVIEW

File: scanpipe/forms.py [hunk 1]
```diff
@@ -117,6 +124,10 @@ def __init__(self, *args, **kwargs):
         name_field.widget.attrs["autofocus"] = True
         name_field.help_text = "The unique name of your project."
 
+    def clean(self):
```
Comment (tdruez) [inline scanpipe/forms.py:127]: No need to override the `clean` method here. You can simply use the `clean_<fieldname>()` Django's API.
https://docs.djangoproject.com/en/3.1/ref/forms/validation/#cleaning-a-specific-field-attribute

**Review by tdruez — CHANGES_REQUESTED** [review]
Thanks for the PR. 
Could you add a unit test along the code changes?

File: scanpipe/forms.py (hunk 1 above)
Comment (Sagarsangwan) [reply scanpipe/forms.py:127]: > Thanks for the PR.
> Could you add a unit test along the code changes?

Yes, will do that shortly.

Comment (tdruez) [reply scanpipe/forms.py:127]: Hi @Sagarsangwan, any progress on this change?

**Comment by tdruez** [comment]
Closing, see https://github.com/nexB/scancode.io/pull/158

**Pull request closed without merging**